lib/stat_tracker.rb

```diff
@@ -34,37 +34,31 @@ def lowest_total_score
     games.min_by { |game| game.total_goals }.total_goals
   end
 
-  def find_home_wins
+
+  def find_game_teams_by_hoa_and_result(hoa, result)
     game_teams.find_all do |game_team|
-      game_team.hoa == "home" && game_team.result == "WIN"
+      game_team.hoa == hoa && game_team.result == result
     end
   end
 
   def percentage_home_wins
```

Comment:
there's an opportunity to also combine the `percentage_home_wins` and `percentage_visitor_wins` methods into a single one that take an argument of `(game_teams_collection)`

here's an example of how I did this with an averages calculation:
```ruby
  def team_averages(sorted_team_scores)
    avgs_by_team = {}
    sorted_team_scores.each do |team, scores_array|
      avgs_by_team[team] = (scores_array.sum / scores_array.count.to_f)
    end
    avgs_by_team
  end
```

Reply:
oy, I just realized that these are the main methods and not helper methods - so obviously we don't want to combine them... my bad! merging now...